templates/download/raspberry-pi/index.html

```diff
@@ -46,12 +46,9 @@ <h1>
     <h2 class="u-sv2">
       Download Ubuntu Desktop
     </h2>
-    <p class="p-muted-heading">
-      Recommended desktop
-    </p>
   </div>
   <div class="row">
-    <div class="col-8">
+    <div class="col-6">
```

Comment:
The two section are not aligned:

![image](https://user-images.githubusercontent.com/2376968/196699325-91436876-fbe9-4aea-9e9f-837c08955ceb.png)

Whereas they are in the Download Ubuntu Server section further down the page. This is quite hacky, but we need a way to keep those sections aligned visually while also having "Recommended Desktop" flow correctly on smaller screens.

```suggestion
    <div class="col-6">
        <p class="p-muted-heading">&nbsp;</p>
```

Reply:
ok. well, I thought the design was not meant to be done this way, but I don't mind getting some design input if needed. For now, I'll update it this way